### PR TITLE
Add taiga smtp setup in readme

### PR DESCRIPTION
**Pull request by mich4xD**

### CODE REVIEW

File: src/project/README.md [hunk 1]
```diff
@@ -36,3 +36,29 @@ Check [Taiga Documentation](https://taigaio.github.io/taiga-doc/dist/) and [Taig
     Run `$ sudo docker ps | grep project` to verify if the containers listed above are up and running. Proceed if no error detected, otherwise run `$ sudo docker logs [container name]` to check the container logs for troubleshooting.
 
 2. Change the default Taiga administrator password after the first login. Use the following user name and password: _admin_ and _123123_ for the initial login.
+
+3. Updated SMTP settings with the information from the SMTP provider in order to be able to add new users to projects:
```
Comment (bahram-aliyev) [inline src/project/README.md]: I would swap `2` and `3`, as the user is still in the console and can change the config file without interruption.

File: src/project/README.md [hunk 2]
```diff
@@ -36,3 +36,29 @@ Check [Taiga Documentation](https://taigaio.github.io/taiga-doc/dist/) and [Taig
...
+    ```
+    ```
+    Remove the '#' and add the correct values in the following parameters at the bottom of the file:
+    DEFAULT_FROM_EMAIL - the email alias for sent emails
+    EMAIL_BACKEND - leave the value as 'django.core.mail.backends.smtp.EmailBackend'
+    EMAIL_USE_SSL - should be True
+    EMAIL_HOST - host of the smtp provider
+    EMAIL_PORT - SSL port of the smtp provider
+    EMAIL_HOST_USER - user from the smtp provider
+    EMAIL_HOST_PASSWORD - password user from the smtp provider
```
Comment (bahram-aliyev) [inline src/project/README.md]: I would suggest creating a bullet list describing only the variables that have to be set for enabling emailing, and something like `Leave the rest of the settings intact.` for informing that default values are good enough.

File: src/project/README.md [hunk 3]
```diff
@@ -36,3 +36,29 @@ Check [Taiga Documentation](https://taigaio.github.io/taiga-doc/dist/) and [Taig
...
+    ```
+
+    After setting up these values, restart the container using the following command:
+   
+    ```
+    $ sudo docker restart project-backend
+    ```
+    
+    Now, when adding users to a project, they will receive an email notification with the invite.
```
Comment (bahram-aliyev) [inline src/project/README.md]: Maybe just ask to create a new user in Taiga and check if the email is received. This way, the user can confirm that emailing works.

Comment (mich4xD) [reply src/project/README.md]: You need to create a new proejct before creating a new user

File: src/project/README.md (hunk 2 above)
Comment (mich4xD) [reply src/project/README.md]: They still need to uncomment these settings, even if the value is default.
\

File: src/project/README.md (hunk 1 above)
Comment (mich4xD) [reply src/project/README.md]: cool cool

**Comment by bahram-aliyev** [comment]
@mich4xD Added a few insignificant changes, have a look, please.

**Pull request merged**